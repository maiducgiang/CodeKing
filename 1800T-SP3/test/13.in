18
82
29573 -97648
50900 -46935
6479 86286
91870 -3298
64039 -31929
79578 85596
20790 51210
5680 95416
81258 12788
90117 75358
30367 80912
37528 -69871
47998 72083
78495 72519
53655 -20884
98013 38468
54373 -51888
9519 7532
25603 83192
52345 84325
66989 -58748
2877 47749
28362 -41984
57857 46229
31887 -25334
83118 31616
51739 -83327
98265 95099
43361 74202
79006 -70977
71178 -6728
6661 73762
74690 -60973
93984 1900
12474 -97113
26041 -4187
93358 46043
56335 30604
62669 26553
61343 11863
4417 -60966
64640 -71777
63583 -77498
23069 85743
87883 -51100
58677 5832
44933 -910
66181 50977
37793 -74467
50894 3737
28784 -78720
7382 -60141
53697 15474
24482 -15706
24368 -70999
85234 -48967
59325 -65129
25216 69137
27617 67523
33130 74152
91221 71015
32392 10538
56570 -22733
93173 92600
5243 36948
43937 -2585
51666 -39347
5810 -17224
69447 67588
79926 55580
71557 27970
85428 -74492
97241 74709
24640 27907
25235 -91423
97568 4969
53576 84890
36752 75580
330 15986
78855 55997
36450 -35930
13500 -54806
97
25876 14788
23026 40783
24451 83366
34743 89383
35549 22092
61295 10318
57003 50630
63587 30432
42768 -90019
36086 -76368
77389 -43471
62766 -59190
31487 84690
1717 41867
90882 40576
23627 94113
22504 52669
8700 -31621
20787 42859
78986 1962
86765 60648
34082 -79848
61549 43706
12967 -82708
260 1249
14903 61195
72112 -89655
37450 83557
31521 -15085
92583 71507
8152 57705
24851 42292
15566 -19280
75456 92436
5118 -97841
14407 -93133
6326 98431
61955 13340
24937 -59889
91361 65842
66457 -77010
62749 -81553
3798 94330
32426 63084
29570 -60121
39496 31133
15966 75575
817 -60669
87215 65858
53304 -10690
65829 -77666
68207 58695
41290 -60507
93026 6794
29488 -69028
36237 22254
50430 18829
68621 -81801
21021 53555
27488 -92384
23378 -19521
33744 -20605
38132 84133
41701 -5077
9547 -14140
73718 86425
60930 -42565
14063 28123
1776 25535
14412 -49229
35016 -3586
43086 56110
62300 21197
51127 -76668
26832 69732
74188 5881
72356 -98520
26233 92199
47371 92333
4265 -42918
4171 74532
11634 98904
5684 -42332
89751 -97122
12143 -17021
99456 61713
1633 59600
37955 -83529
9468 -55535
40618 -88757
73109 87464
53622 84402
12609 -8894
24447 87349
20129 39902
39768 12003
47969 -18215
73
40768 -87052
48695 -70455
74487 -76507
24567 -53327
45142 29124
61718 32744
45747 18204
96293 -77467
23237 -68094
50479 -69247
68519 78365
99959 12469
1732 -71456
86966 24440
57244 -47235
10885 -72466
29210 -69743
27451 49463
74461 -77440
48263 -21091
42460 -71251
38731 15599
89247 52442
33102 -63484
97809 -59296
29513 41667
83911 -24077
14836 1174
83499 -10480
76112 -11383
47839 62253
3513 -40013
175 95189
16107 42616
64499 67269
66560 30955
50161 -36676
28726 -43556
98081 -64977
4101 36435
92635 12135
48174 -29231
7074 -55238
50642 -63113
70880 -60367
58747 45042
93409 -47940
61599 37167
42935 -72110
35255 31631
91104 -65753
31560 -91853
60763 -8283
15980 35616
68446 33528
27458 28388
68112 12013
75994 72054
27033 -51916
29051 -84684
31836 -54057
47401 93148
49782 77726
13291 -83624
40876 33412
66923 58946
91012 -82527
98794 11970
43935 50554
36354 58480
73955 70085
74552 -61409
4140 32862
54
33708 16761
99116 61304
78312 35329
33519 -76967
4518 3347
86422 -73533
48395 -75679
34143 -62307
91844 63070
55660 -30044
40873 60722
85461 -98329
5788 86951
30232 -97980
38801 48389
38257 23934
25395 -22394
11978 -40695
10524 13082
35250 48415
9211 -48933
14901 80622
79388 33684
44298 60444
65404 28876
30667 19268
69614 -44002
47620 59377
31113 -27821
14440 -28820
82941 -66147
38422 -54336
69602 -42328
28358 -98452
97994 48921
60297 38928
9447 3570
43908 -60759
85840 2467
57769 -10115
60278 -27213
77673 26474
76972 -87131
96084 11998
56412 -469
61172 -75243
31916 -22202
69035 85041
88609 -47089
18770 -20310
30671 2138
41562 -69407
12722 -43911
73873 -31368
99
85314 -91948
87514 -33511
27271 90141
44650 55172
76667 94077
28940 53963
71999 -13358
78163 -58307
20442 63423
4334 5262
9673 6691
47701 -18948
28177 15382
49003 54475
58061 -73390
26744 36926
75706 46419
89500 80221
47637 -80458
78802 -53787
109 -67210
29681 -2227
46052 -96567
60939 29085
58912 -50433
41102 -13039
65486 39104
92444 45044
22659 -36393
11866 78284
99638 18881
37939 14049
29165 -24366
53496 86003
62554 63669
27423 -24663
67273 -68907
47562 89401
97173 27528
71466 -38221
61566 -28719
97264 -22322
6556 57609
65710 3185
88434 83272
26858 21689
23392 24452
20917 36272
66742 -27379
57907 -56887
53897 69202
1364 -67903
29684 -53119
52797 -90153
3737 -5504
34659 75263
86129 67639
71258 79536
79498 -98552
8578 53441
36381 98767
13873 -92688
24667 -8809
31648 -94880
26220 -88977
57920 -64215
19325 -55700
91166 55210
67184 61962
70811 -857
23519 83570
96676 42080
27151 -55956
96678 11836
6872 -98176
37039 -84533
92032 -7259
2881 93559
34703 -71189
72175 20228
38980 -26471
48232 -30677
27255 90055
12480 63759
10219 88323
23000 10469
94530 -67687
32856 -18580
31410 88414
69426 -55236
54706 19883
54450 80236
32493 -5819
50596 -61956
81984 -59114
29375 -41966
60023 17573
66909 71436
40534 -32567
85
223 -71689
26642 -27037
70579 -98286
51885 37873
85166 -22188
92522 -14263
25500 99440
67875 32439
5611 -18664
17216 58859
18853 54386
84017 62836
60637 -4218
4218 -72920
43328 66715
97909 91265
75354 -53931
72487 -9395
4040 -83493
47563 21435
75161 -25376
76671 40098
9540 12522
43998 -24036
33444 -74900
13663 -86980
51317 -5941
20945 91569
35566 -23461
67631 72267
74149 -2315
95472 -43298
46969 -34660
52364 89374
6221 71440
72631 -22351
43992 92229
56098 -75731
78486 -72118
53326 12038
9196 -64371
74282 -74613
67934 27583
37440 -91457
25050 -70431
87907 -77278
89232 -42081
78013 78479
50975 60312
36141 60701
15936 13504
13332 -37983
7578 -73512
4763 -11845
47915 -69719
39703 -73667
94168 -68747
46540 -90064
78375 -35442
82787 50855
91971 80977
58442 11836
18257 93740
93576 57806
27170 40263
15815 -97593
35303 -79049
80199 41645
26014 30504
11651 10459
40191 23971
97121 -98106
22856 89521
28762 33304
82462 -54521
78229 89038
75431 -73946
26781 4291
94094 21218
1147 41225
9549 -26959
90306 85908
37920 -45119
47523 -86609
69833 34295
100
52782 69123
37683 25451
30489 -44356
57634 85212
15255 -67964
84094 28630
27754 -93090
14001 -82819
59766 56945
1891 39911
66642 34266
69615 -10024
32242 -44876
81893 66733
40804 -15952
99953 46873
89179 71360
67613 37089
65598 29746
52348 42416
73758 -50819
4689 -97205
91487 -22064
92183 -94337
8809 52002
95692 -89444
10713 54646
51307 4273
74553 -55749
37912 -39044
69534 -75721
77842 28806
15368 -65955
84249 -195
71270 74308
83800 11122
25062 37053
80871 6235
56309 -17247
91511 47571
73164 88920
50944 78412
96160 42560
34687 -42411
58788 98540
27558 10116
26001 86151
70314 -38945
85807 28008
15231 -56527
83935 -89895
66700 36174
2991 -53150
82603 48884
39295 14756
66506 -40887
6338 -99766
42268 -1443
6214 -77648
89095 92451
82566 -93328
84370 -1184
66240 1485
38542 33866
97611 69182
72907 -82960
88822 -44826
11697 27364
69901 15852
46617 69814
85704 -3818
60472 12421
89376 20227
94515 -38618
63142 -89634
61932 -86537
26682 6831
67550 -8567
17510 -84855
91624 59829
92348 90972
67387 -94956
46349 22626
39398 57421
93059 75276
1994 19665
96402 -14981
17160 8208
54087 -15575
31701 11071
61210 1543
12436 -10613
29183 10918
49440 -83685
75154 93974
34406 36119
55687 -78713
71784 74390
60099 41465
71116 -65967
67
43168 -64161
1656 75368
72046 -40665
45580 89697
29948 -64957
71679 -35981
26889 -69342
46439 -68740
91577 -47902
54692 -19747
72865 83793
62013 87017
27249 75710
78453 -82971
49524 94225
71705 -9931
69800 25472
40361 12652
27260 -92537
18944 21497
92364 -76054
60663 -51035
74029 -44101
12349 19823
36431 -30997
89146 -2747
78165 -89404
2497 24943
36488 -91446
38729 -12799
93110 15801
8115 54147
40814 -68013
94528 -20425
39326 -30796
13650 47820
22244 26014
73967 -31747
2236 30308
2268 -87681
84739 -81700
6408 -82091
54039 -90244
74044 61297
31728 59917
76661 48584
37063 -96879
25592 -35459
17187 -14199
45762 -53220
96702 89046
32048 -79586
16652 76755
35761 17094
67347 10614
81401 -13505
64291 38042
13444 -8992
20964 -20053
69197 98932
53318 -87777
86789 -67482
85987 6079
60522 -74625
35340 -18456
28945 92595
62234 -36996
98
13598 98301
4365 49775
61614 35678
61796 -33542
42102 52804
31314 -18974
18511 25464
36134 -38984
92498 -46199
2860 -46113
50032 -21021
64115 -30730
97405 76015
27599 -10430
97551 -5989
2218 -8243
59488 -8559
92904 -24026
74132 -9573
61107 -56165
78338 70361
86177 -79932
73950 -12926
19226 71455
12789 42651
24967 -40897
60823 -73678
12644 -43764
23458 46086
5378 -49772
63509 9249
60172 54807
98533 12421
48907 61879
16014 -43274
70822 -9091
16446 50877
2833 39042
56445 -61606
78411 57127
26497 94631
53047 98554
79438 -9243
65836 37125
43047 -70955
47295 10750
76424 54596
88554 30315
3605 -45555
71858 21777
109 37434
15493 -58168
10775 16661
1697 95134
95255 -48472
81036 91052
49366 -95911
94580 -61535
1916 -83031
8437 49659
74557 96894
77331 62533
32534 -18454
84680 -27902
87804 -59766
75993 -16025
53062 57535
54491 -5164
55467 -35471
93564 58771
28473 -77796
62852 30745
48817 -8904
88258 -18567
16034 96238
83507 -79066
65324 10090
41081 39764
50565 -51105
16951 53788
67247 -16517
31096 95310
71295 -85705
72151 18229
40237 53464
16894 -41250
39901 -83552
46891 -75331
50313 62042
32994 -94006
93887 -71067
60293 -93063
47287 16493
41240 48202
24311 -34597
87393 -13992
39564 60098
31981 67136
37
28976 85974
47859 -96615
99780 33425
18650 47001
89480 29813
80531 9146
12863 -34824
4469 76779
1910 -24825
8422 -3424
45687 3809
28704 -30776
27708 75086
57465 6391
32456 30013
53760 12001
94171 8001
3629 14269
4147 78047
43310 93379
78396 93105
52570 5773
34572 -87522
88432 -16830
28873 18193
4067 -54358
71733 63780
5820 63589
35509 10175
29218 40628
38307 53636
88002 -65691
82289 78648
34005 -93837
82595 64326
90261 32371
34404 82199
96
14422 19578
36146 81994
2196 -57939
36129 -22238
95372 -73305
80463 70903
82845 93910
89431 -8493
38000 34099
95809 -36680
60874 -76206
34902 85516
92935 89196
22824 15248
6295 -98211
39077 38107
93412 39318
1610 -99504
1166 -46492
83416 -41052
8677 89038
24971 14526
94328 -96267
64377 -17212
28987 -13966
68621 -34008
48996 -7626
17177 92930
36153 89760
57017 26070
3222 -85877
67806 70124
39025 -80365
44355 -3474
93739 11665
94725 -10914
24552 87005
17191 -16528
48581 -20292
96216 75773
37355 -42703
53862 136
91760 92141
60209 -68751
51596 37040
18342 12008
84579 -63720
5959 93454
29986 -93942
8317 -95654
39816 40548
47209 74543
82366 -47976
40807 -35556
14101 -28695
5287 -70818
72175 -85288
76257 27123
71805 17318
27854 -38940
7157 60031
24823 -26080
75661 -32180
1037 34090
19956 -98546
2371 -89505
44922 -82153
14863 -8068
33000 59383
55391 -6457
71772 -51106
71348 -22078
5355 -17159
46873 -84366
30780 95298
37070 75710
80719 -76015
79731 -91632
48097 65067
74988 -15474
43414 8124
50434 25510
48891 93897
84871 -58037
8192 -82015
25634 -14993
17819 49330
87078 -86800
99995 -49198
31023 90500
79362 -27822
45467 -35591
47123 -45591
97775 -20106
22579 -76855
83734 -3102
31
10000 37590
55137 72031
99976 -34141
98065 -62302
74677 -31841
23778 -97741
1013 -87764
24891 405
51506 30429
53873 -43908
13974 -45183
90594 24414
10109 5349
83879 64525
48415 55213
83549 23244
52557 47541
391 68215
40803 -49823
59656 -47497
23483 -32437
65263 9423
6300 33466
6094 15543
84541 -47495
48397 -66452
19656 -85252
25407 67513
21591 -73301
7593 -84833
48050 -68315
16
35297 12642
64332 -52485
84508 -99920
61435 -88087
73676 -3288
6188 -15317
14569 -4199
53335 -9790
80601 15613
83852 90134
54769 47917
26730 26484
26560 -21492
96792 40702
3386 35545
91176 -98204
80
79031 95351
79883 -8589
59935 -4861
22840 -76766
43963 4138
10193 98892
92372 -37163
79440 -91629
81715 54401
48690 -60543
98033 -40779
75390 18135
11940 36888
29411 -72624
19558 68755
47915 -63463
8792 -6501
83569 -98780
87882 7047
16458 -4222
18600 72066
94018 -44275
36365 56476
84065 44534
24500 -4097
54102 88147
34534 -7611
92321 12229
28191 -52457
71095 8549
51506 6871
96744 71951
82646 24926
11477 21552
65700 66167
3035 -97995
11990 96884
48245 13720
46293 -57237
74285 -96254
57678 -98999
75660 -2105
23277 23754
5642 -57136
22323 74149
95760 72323
7480 -14743
23971 -14169
36173 -18510
53006 -85391
14491 -10370
86290 63179
10604 -74512
90911 -93968
34598 41091
82615 3729
13425 94068
77704 40594
52023 74411
15279 -90430
15012 -30875
70882 -45931
96894 77416
35902 50652
25154 73468
48652 -44747
44203 68248
50731 11430
62901 41503
74126 46597
62009 21282
30327 40737
1700 -45124
29326 78066
7680 -42436
96445 -48345
60007 66355
29552 72915
42742 -66840
71460 61217
60
4556 287
1509 -59938
47009 8174
28027 32757
61620 -5272
90106 -71591
66919 48346
54151 -91076
65798 67081
75281 2092
9848 -76239
29621 -53864
93578 20126
14634 33356
49414 -93683
65832 57363
98108 -22058
68413 63522
85387 34160
35324 -84869
19039 -32574
6574 44845
45996 3386
91413 207
88338 9661
78800 -40442
40512 -97407
48173 -43901
72141 -23513
62894 9637
28376 -15034
37647 68597
22863 69836
30380 -6336
90087 -45658
21315 82381
97540 -57077
3148 71669
4934 -64745
58267 66610
71775 22402
41777 -64345
72142 28177
32701 17568
59226 26782
76826 -85928
28111 1237
9288 -45267
78757 -90964
68048 86070
37159 87294
37910 16435
32322 -18718
47055 -67584
57319 45014
22189 -254
66181 18764
89437 -28065
18718 -68210
38455 68757
16
86032 -41732
35751 -84754
14855 -90386
6464 32932
98750 -32875
42329 96517
39886 81277
17042 89793
17824 73089
3693 -80629
36333 -45778
96568 -32741
65646 -83050
10095 37314
45130 -22814
84018 84051
97
59190 51711
60820 -63704
66959 66464
24785 -28192
51275 -10405
33877 -48939
79599 -69388
39338 25934
73997 -73100
38554 40653
94446 -78529
82416 19304
99553 64256
39988 -23759
12406 -10339
86601 46619
31917 23851
65623 -6808
19065 -35484
36699 33953
51649 -66352
83801 -6178
68126 -53821
82883 -8947
81163 -8565
49057 75475
60515 43095
38759 64490
1690 12386
38880 47082
98243 90882
39297 -96035
63554 -19526
38759 80598
70404 -91743
96289 97963
47190 -3176
98147 -30
77700 -12554
19606 -22969
70180 11422
27558 -14611
90502 99776
67998 76949
41146 81495
84615 11432
57035 -37624
26506 5537
62741 96380
35711 69908
17613 3702
47038 -27684
37576 -32318
58418 -43336
11671 -40394
13475 -7290
1856 -44561
82306 -51596
708 -13202
407 68197
79417 -38497
5452 -6451
37044 44424
11307 27734
74314 56240
70890 38343
78489 -23327
20241 88658
56511 69242
41992 -52503
58326 -24099
35111 -44877
31795 41198
84872 72576
27562 -63884
40769 13911
83965 -53250
26445 57410
91467 60410
51315 -13441
81804 -27990
78141 47297
71080 85238
90072 -578
10403 -87877
21633 71325
4050 56168
34531 72518
36450 74731
69357 62141
38538 -45998
92454 -8919
91677 -13831
42352 -28217
79327 -44789
46244 37261
87503 89958
96
90690 -1319
33877 -39454
17115 -60853
38870 -36051
92034 -24044
55702 -41642
66284 37474
19079 -81755
35170 -29490
4914 40541
64637 42747
48042 -16378
31465 -28959
38851 87387
10126 9008
38348 32207
47593 -61613
43798 73212
11982 -52436
44071 -52411
82899 76789
38374 40797
94474 85830
18514 -91475
80597 17583
35581 49900
86969 -1699
49797 19411
32520 38012
81684 46086
25232 57233
41545 -43334
51659 32202
93830 -78143
48595 88423
87372 -42886
42125 57546
93234 -39569
76672 -32924
85373 -86194
38983 54529
42908 -1015
80589 -38341
93306 65173
75534 7034
26913 81947
65817 -25880
23402 2599
93376 72731
34973 -69403
36562 47998
13675 -94634
95632 -4485
25837 8134
72992 18999
8559 -44542
67923 14552
47002 -53988
17735 55711
32418 79677
19824 3733
88376 60864
27290 -97238
20138 -90861
16162 -46511
66043 7955
38442 -7666
81040 -31162
82784 63936
17525 80174
4900 -12230
80743 39877
32947 -29875
68445 -35462
34691 75400
67151 -71819
99943 814
80077 -57487
62705 -27568
59728 1263
46454 49297
17554 -87538
61736 -88340
41559 -92457
5836 -34386
90434 55084
12056 72151
10955 85726
33650 51763
22254 -39647
74000 29406
91164 -95787
11703 13109
5098 92577
56003 22589
19108 11585
