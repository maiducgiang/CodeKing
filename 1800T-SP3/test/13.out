12
14
9
12
14
7
19
13
14
12
11
2
4
12
8
2
21
22
